src/Biscotti/Cookie/Parser.purs

```diff
@@ -100,8 +100,15 @@ parseAttribute =
 parseSimpleCookie :: Parser Cookie
 parseSimpleCookie = do
   name <- stringWithout ([ ';', ',', '=' ] <> whitespaceChars) <* string "="
-  value <- stringWithout ([ ';', ',' ] <> whitespaceChars)
+  value <- try stringLiteral <|> stringWithout ([ ';', ',' ] <> whitespaceChars)
   pure $ Cookie.new name value
+  where
+  stringLiteral :: Parser String
+  stringLiteral = do
+    _ <- char '"'
+    x <- stringWithout ['"']
+    _ <- char '"'
+    pure ("\"" <> x <> "\"")
```

Comment:
Actually, should we keep the double quotes in the end result or just keep `x`? Now I feel like later makes more sense.

Reply:
Nvm, they have to be there. Or we have to add double quotes when encoding a string value with white spaces in it. Otherwise the encode-decode or decode-encode roundtrip would fail.

Reply:
I would think we should preserve the original value. Most likely if there are double-quotes, they're needed for further parsing. E.g. the value could be JSON.

Reply:
Yeah that's true.